model-optimizer/extensions/middle/UpsampleToResample.py

```diff
@@ -141,7 +141,8 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
         mul.out_port(0).connect(interpolate.in_port(1))
         axes_node.out_port(0).connect(interpolate.in_port(3))
 
-        scales_node = Const(graph, {'name': upsample_name + '/scales', 'value': factor_value}).create_node()
+        scales_node = Const(graph, {'name': upsample_name + '/scales',
+                                    'value': factor_value.astype(np.float32)}).create_node()
```

Comment:
For the sake of unification instead of `astype` I suggest to add `float32_array` from the same place as `int64_array` .
`from mo.front.common.partial_infer.utils import float32_array`

Reply:
Done.